Mathlib/Algebra/BigOperators/Basic.lean

```diff
@@ -2008,7 +2008,7 @@ open Finset
 /-- `Fintype.prod_bijective` is a variant of `Finset.prod_bij` that accepts `Function.bijective`.
 
 See `Function.bijective.prod_comp` for a version without `h`. -/
-@[to_additive "`Fintype.sum_equiv` is a variant of `Finset.sum_bij` that accepts
+@[to_additive "`Fintype.sum_bijective` is a variant of `Finset.sum_bij` that accepts
 `Function.bijective`.
 
 See `Function.bijective.sum_comp` for a version without `h`. "]
```

Comment:
Do you think we should rename this to `finset_sum_comp` to match?

Reply:
I think they're similar and will get discovered through each others' names, which IMO is good enough. Maybe we should point out that `sum_comp` is for the case where `h` is `fun t => rfl`.

Reply:
ah, that's what the "without `h`" means. I wonder if this one could get away with having `e` be implicit.

Reply:
@eric-wieser just to triple-check you're happy with the current state before I `r+`